spec/features/agents/agent_can_see_users_rdv_spec.rb

```diff
@@ -23,14 +24,22 @@
   end
 
   context "with one RDV" do
-    let!(:motif) { create(:motif, organisation: organisation, service: service) }
     let!(:rdv) { create :rdv, :future, users: [user], organisation: organisation, motif: motif, agents: [agent] }
 
     before do
+      %i[seen excused revoked noshow].each do |status|
+        create(:rdv, :past, status: status, users: [user], organisation: organisation, motif: motif, agents: [agent])
+      end
+
       visit admin_organisation_user_path(organisation, user)
     end
 
     it do
+      expect(page).to have_content("Excusé\n1 RDV")
+      expect(page).to have_content("Vu\n1 RDV")
+      expect(page).to have_content("Annulé par un agent\n1 ")
+      expect(page).to have_content("Non excusé\n1 ")
+
```

Comment:
Je suis d'accord avec toi ici : non seulement les perfs sont meilleures quand on met les `expect` dans le même `it`, mais c'est aussi bien plus lisible !